src/libutil/addr.c

```diff
@@ -1060,6 +1060,21 @@ rspamd_inet_address_connect (const rspamd_inet_addr_t *addr, gint type,
 
 	if (addr->af == AF_UNIX) {
 		sa = (const struct sockaddr *)&addr->u.un->addr;
+
+		if (type == (int)SOCK_DGRAM) {
+			struct sockaddr ca;
+
+			memset (&ca, 0, sizeof(ca));
+			ca.sa_family = AF_UNIX;
+
+			r = bind (fd, &ca, sizeof (sa_family_t));
+			if (r == -1) {
+				close (fd);
+				msg_info ("client autobind failed: %d, '%s'",
+						errno, strerror (errno));
```

Comment:
```suggestion
				msg_info ("unix socket client autobind failed: %s, '%s'",
						addr->u.un->addr.sun_path, strerror (errno));
```

It is way better to see a socket name than errno code in my opinion.